packages/console/src/pages/SignInExperience/components/ManageLanguage/ManageLanguageButton.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+
+import Button from '@/components/Button';
+
+import LanguageEditor from './LanguageEditor';
+import * as styles from './ManageLanguageButton.module.scss';
+
+const ManageLanguageButton = () => {
+  const [isLanguageEditorOpen, setIsLanguageEditorOpen] = useState(false);
+
+  return (
+    <>
+      <Button
```

Comment:
can we remove this button from the "editor" component?

Reply:
Reorganized.